lldb/test/Shell/helper/toolchain.py

```diff
@@ -22,6 +24,55 @@ def _disallow(config, execName):
     config.substitutions.append((" {0} ".format(execName), warning.format(execName)))
 
 
+def get_lldb_args(config, suffix=None):
+    lldb_args = []
+    if "remote-linux" in config.available_features:
+        lldb_args += [
+            "-O",
+            '"platform select remote-linux"',
+            "-O",
+            f'"platform connect {config.lldb_platform_url}"',
+        ]
+        if config.lldb_platform_working_dir:
+            dir = f"{config.lldb_platform_working_dir}/shell"
+            if suffix:
+                dir += f"/{suffix}"
```

Comment:
I was going to be pedantic and ask for os.path.join here but of course that would make paths in the host format not the remote's, correct?

Reply:
Though I would default suffix to `""` instead of `None` just to be clear that it's default is no suffix but if you do want a suffix, it should be a single string.

Reply:
Replaced it with posixpath.join (since we're doing that in case of remote-linux) for consistency.
Fixed suffix default value.